language/lambda_spec.rb

```diff
@@ -274,6 +274,20 @@ def obj.define
     lambda { }.lambda?.should be_true
   end
 
+  it "requires a block" do
+    lambda { lambda }.should raise_error(ArgumentError)
+  end
+
+  context "with an implicit block" do
+    before do
+      def meth; lambda; end
+    end
+
+    it "can be created" do
+      meth { 1 }.().should == 1
```

Comment:
>use call instead of () here, it's much easier to read in this case.

This was intentional, to try and remain consistent with how these things are `call`ed elsewhere.